src/Shared/ResourceUtilities.cs

```diff
@@ -171,6 +172,13 @@ internal static string FormatResourceString(out string code, out string helpKeyw
             return ExtractMessageCode(true /* msbuildCodeOnly */, FormatString(GetResourceString(resourceName), args), out code);
         }
 
+        [Obsolete("Use GetResourceString instead.", true)]
```

Comment:
Note, new code start here.

Reply:
Wait, how does this work?  Shouldn't this method at least have a single `string` parameter?

Reply:
Whoops, I must have removed the string parameter. I used this method to find the original usages so looks like I must have removed it when I added the obsolete.